test/cypress/e2e/data/mockDeploy.ts

```diff
@@ -0,0 +1,10 @@
+export const mockDeploy = {
```

Comment:
Could you please update this to a function that returns this object? We should get in the habit of only assigning variables for mocks this way.